release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3054,6 +3054,40 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.18.27
+[id="ocp-4-18-27_{context}"]
+=== RHSA-2025:19047 - {product-title} {product-version}.27 bug fix and security update
+
+Issued: 29 October 2025
+
+{product-title} release {product-version}.27 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:19047[RHSA-2025:19047] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:19045[RHBA-2025:19045] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.18.27 --pullspecs
+----
+
+[id="ocp-4-18-27-bug-fixes_{context}"]
+==== Bug fixes
+
+* Before this update, if the OVN-Kubernetes controller was not processing updates from the Kubernetes API server and configuring the open virtual network (OVN) databases on each node, then the OVN-Controller, which consumed this database, might have connected to the database before the OVN-Kubernetes controller had configured them. As a consequence, the OVN-Controller synced with a stale OVN database, consumed source network address translations (SNATs) that were configured to support the egress IP, and proceeded to the gratuitous address resolution protocol (GARP) for the associated IP even though that IP might have moved to another node. With this release, these GARPs are blocked when the OVN-Kubernetes controller is not processing updates.  (link:https://issues.redhat.com/browse/OCPBUGS-62671[OCPBUGS-62671])
+
+* Before this update, the Cluster Version Operator (CVO) in 4.19.9 and 4.18.23 started to require bearer token authentication in metrics requests. As a consequence, HyperShift and Hosted clusters were broken because the metrics scraper did not provide client authentication. With this release, the CVO does not require client authentication for metrics requests. As a result, access to CVO metrics scraping is recovered on HyperShift and Hosted clusters. (link:https://issues.redhat.com/browse/OCPBUGS-62869[OCPBUGS-62869])
```

Comment:
AFAIK "HyperShift" is not a real product name.

@lahinson is this more likely correct? The bug itself refers to ROSA HCP and I am not sure how all of these things interact :sweat_smile: 

```suggestion
* Before this update, the Cluster Version Operator (CVO) in 4.19.9 and 4.18.23 started to require bearer token authentication in metrics requests. As a consequence, {hcp} clusters were broken because the metrics scraper did not provide client authentication. With this release, the CVO does not require client authentication for metrics requests. As a result, access to CVO metrics scraping is recovered on {hcp} clusters. (link:https://issues.redhat.com/browse/OCPBUGS-62869[OCPBUGS-62869])
```